src/modules/75-cf/pages/onboarding/views/ValidatingYourFlagView.tsx

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { useEffect, useState, useRef } from 'react'
+import { Container, Text, Heading, Layout, Icon } from '@harness/uicore'
+import { useParams } from 'react-router-dom'
+import { Color, FontVariation } from '@harness/design-system'
+import { Classes, Switch } from '@blueprintjs/core'
+import { String, useStrings } from 'framework/strings'
+import type { PlatformEntry } from '@cf/components/LanguageSelection/LanguageSelection'
+import { ApiKey, FeatureFlagRequestRequestBody, useGetAllFeatures, GetAllFeaturesQueryParams } from 'services/cf'
+import { useToggleFeatureFlag } from '@cf/hooks/useToggleFeatureFlag'
+import { CF_DEFAULT_PAGE_SIZE } from '@cf/utils/CFUtils'
+import { ResourceCenter } from '@common/components/ResourceCenter/ResourceCenter'
+import css from './ValidatingYourFlagView.module.scss'
+
+const POLLING_INTERVAL_IN_MS = 30000
+const INITIAL_INTERVAL_IN_MS = 240000
+
+export interface TestYourFlagViewProps {
+  flagInfo: FeatureFlagRequestRequestBody
+  language: PlatformEntry
+  apiKey: ApiKey
+  environmentIdentifier: string | undefined
```

Comment:
Might as well just mark as optional
```suggestion
  environmentIdentifier?: string
```